### DISCUSSION TITLE
How to activate feature 'copy code' feature

### DISCUSSION THREAD

**Discussion by LarryMarzanJr** [dillonzq/LoveIt#770] [Q&A]
Hello,
As described in README file, https://themes.gohugo.io/themes/loveit/#why-choose-loveit
One of the feature is  'Copy code to clipboard with one click'

I'm already migrated to this theme but my code block doesn't seem to show the copy code button.
https://jpst.it/30i5J

How do I activate this?

**Comment by smit-io** [comment]
I have the same issue. Did you find any solution?

**Reply by LarryMarzanJr** [reply]
The easiest thing I did was to pull the project itself and editing the yaml/toml file. While all my old posts were moved manually into that project. That way the 'copy to clipboard' works perfectly 😊
Not so neat, but it does the job 👍

**Reply by Bert-JanP** [reply]
Hi,

What did you edit in the yaml/toml file to make this work?

```
    [params.page.code]
      # whether to show the copy button of the code block
      # 是否显示代码块的复制按钮
      copy = true
      # the maximum number of lines of displayed code by default
      # 默认展开显示的代码行数
      maxShownLines = 50
```
The settings above are are enabled in my toml file, but I got the same results as from your screenshot.
